src/py/flwr/client/clientapp/app.py

```diff
@@ -66,11 +66,11 @@ def flwr_clientapp() -> None:
             "flwr-clientapp does not support TLS yet.",
         )
 
-    log(INFO, "Starting Flower ClientApp")
+    log(INFO, "Starting flwr-clientapp process")
     log(
         DEBUG,
-        "Starting isolated `ClientApp` connected to SuperNode's ClientAppIo API at %s "
-        "with token %s",
+        "Starting isolated `flwr-clientapp` process connected to SuperNode's "
```

Comment:
I think "isolated" here just means running ClientApp in a process, right?
```suggestion
        "Starting `flwr-clientapp` process connected to SuperNode's "
```